src/pages/home/report/ReportActionCompose.js

```diff
@@ -196,6 +201,12 @@ class ReportActionCompose extends React.Component {
         this.setState({isMenuVisible});
     }
 
+    updateSelectionOnAndroid(text, selection) {
+        if (getPlatform() === 'android') {
+            this.textInput.setNativeProps({text, selection});
+        }
+    }
+
```

Comment:
We're avoiding using platform specific code such as this, please check out 

https://github.com/Expensify/Expensify.cash#philosophy
under `5. Cross Platform 99.9999%` and point `iii`

Reply:
Got it and resolved it. Please have a look.